variables.tf

```diff
@@ -75,6 +75,24 @@ variable "cloudwatch_schedule_expression" {
   description = "AWS cron schedule expression"
 }
 
+variable "cloudwatch_log_group_create" {
+  type        = bool
+  default     = true
+  description = "Whether to create the new Cloudwatch log group for ECS task"
+}
```

Comment:
Keep default ecs task log group behavior, e.g. default value should be `false` and thus no BC will be introduced.